Dockerfile.base

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:16.04
+
+RUN apt-get update -y -qq
+RUN apt-get install -y -qq \
```

Comment:
I didn't hit those as issues when getting the app running. Do you know if they're needed for something today? Easy enough to change the base container when needed.